Dockerfile

```diff
@@ -22,4 +19,8 @@ RUN COMPOSER_ALLOW_SUPERUSER=1 composer install --prefer-dist --optimize-autoloa
     && rm composer.json \
     && rm composer.lock
 
+RUN mkdir /root/project
```

Comment:
`RUN mkdir /root/project` call order has no special meaning, while it must be always before the `WORKDIR /root/project` declaration. `WORKDIR /root/project` must be set after `WORKDIR /usr/src/gush` since the working directory (`pwd`) is attached at `/root/project` and all the operations performed by gush must be executed on that directory.